druid/README.md

```diff
@@ -0,0 +1,122 @@
+# Agent Check: druid
+
+(TBD Add Druid Dashboard Image here)
+
+## Overview
+
+The Datadog Agent collects many metrics from _Druid_ via DogStatsD. Those Druid metrics are related to queries, ingestion, and coordination:
+
+* Query Metrics
+* SQL Metrics
+* Coordination
+* General Health
+* Sys
+
+And many more. See details [here][1].
+
+I addition to metrics, the Datadog Agent also sends a service check for Druid health.
+
+## Setup
+
+### Prerequisite
+
+Druid 0.16 or above is required for this integration to work properly.
```

Comment:
Druid 0.16 is not release yet, but should be soon (maybe sept 2019). Druid Integration should not be public until it's released, it contains upstream improvements for Druid `statsd-emitter` extension.